Add example app for authorization APIs (#399)
heremaps/here-data-sdk-typescript: docs/examples/here-data-sdk-aaa-example/README.md

> # Simple Authorization WEB App 

Reviewer: ```suggestion
# Authorization Web App 
```

> This is a simple Web App with configured compiling TS to JS and bundling with webpack.
> The app shows how to use APIs from `@here/olp-sdk-dataservice-api`; 
> 
> All function from AuthorizationAPI can be used in the same way as used in this example.

Reviewer: ```suggestion
It is a web app with configured compiling of TS files into JS files and bundling with webpack.
The app shows how to use the APIs from `@here/olp-sdk-dataservice-api`.

You can use all the `AuthorizationAPI` functions in the same way as in this example.
```

> ## Setup
> 
> 1) Download the current folder to your PC.
> 2) Add to your enviropment app key and secret.
> ```
> export OLP_APP_KEY=<your app key>
> export OLP_APP_SECRET=<your app secret>
> ```
> 3) Install dependencies.
> 
> ```
> npm install
> ```

Reviewer: ```suggestion
## Set up

1. Download the current folder to your PC.
2. Add your app key and secret to your enviropment.
    ```
    export OLP_APP_KEY=<your app key>
    export OLP_APP_SECRET=<your app secret>
    ```
3. Install the dependencies.

    ```
    npm install
    ```
```

Author: Set up - ignored, Setup should be used.

> ## Build and run
> 
> 1) Compile TS to JS.
> 
> ```
> npm run compile
> ```
> 
> 2) Bundle with webpack.
> 
> ```
> npm run build
> ```
> 
> 3) Start web server.
> 
> ```
> npm start
> ```
> 
> 4) Open in your favorite browser http://localhost:8080

Reviewer: ```suggestion
## Build and run

1. Compile TS files into JS files.

    ```
    npm run compile
    ```

2. Bundle with webpack.

    ```
    npm run build
    ```

3. Start the web server.

    ```
    npm start
    ```

4. Open the web server in your favorite browser: http://localhost:8080
```

Author: Open the web server in - not correct, ignored that.

Reviewer: Please change the formatting.
